src/assets/toolkit/styles/components/message.css

```diff
@@ -0,0 +1,38 @@
+/** @define Message */
+
+:root {
+  --Message-padding: var(--space-xs) 2.5em;
+  --Message-background: var(--color-gray);
+  --Message-offline-background: var(--color-orange);
+  --Message-offline-color: var(--color-white);
```

Comment:
Did you test the contrast of the white on orange using [tota11y](http://khan.github.io/tota11y/) or some other tool?